pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -452,7 +472,7 @@ public String getDefaultValue() {
    * @return evaluated value for this connection parameter
    */
   public String get(Properties properties) {
-    return properties.getProperty(_name, _defaultValue);
+    return new org.postgresql.util.ExpressionProperties(properties).getProperty(_name, _defaultValue);
```

Comment:
This would have an unexpected consequence if someone has `dfwe${...}asdfa` pattern right inside a password property.